brainstormforce/astra
# fix: Menu typography issue with old Header

### Description
Task 1 - [Link](https://trello.com/c/5SEQU1Kf/1017-issue-old-header-menu-typo-with-new-header-menu-font-size-conflict)
Task 2 - [Link](https://trello.com/c/36IB5g5z/1041-individual-menu-component-typography-not-working)

### Types of changes
Bugfix (non-breaking change which fixes an issue) 

### How has this been tested?
Case 1:

- Go to previous stable versions. There set a font family or weight to the primary menu
- Switch to the latest versions and use the new header footer builder, now go to the primary menu and try updating the font family for that, it will not reflect on the front end.

Expected case: The menu font family should be applied 

What's happening now: The typography which is applied in the previous versions cannot be changed at all.
_______

Case 2

- Enable Old Header and Footer
- Set Font Size for Primary Menu - 18px
- Switch to new Header.
- Add Secondary Menu
- Set font size 15px.
- You will notice the issue is 18 px is still applied even when new header is enabled which has 15px font size.


### Checklist:
